src/main/resources/templates/client-create.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en" xmlns:th="http://www.thymeleaf.org">
+<head>
+    <meta charset="UTF-8">
+    <title>Add client:</title>
+</head>
+<body>
+<header>
+    <nav th:replace="_components :: navigationBar"></nav>
+</header>
+<div class="client-create-from-container">
+    <form method="post" class="client-form" th:action="@{/admin/clients/create}" th:object="${clientForm}">
+        <p>Input the following information:</p>
+        <label>
+            First Name
+            <input type="text" th:field="*{firstName}">
+        </label>
+        <p></p>
+        <label>
+            Last Name
+            <input type="text" th:field="*{lastName}">
+        </label>
+        <p></p>
+        <label>
+            Email
+            <input type="text" th:field="*{email}">
+        </label>
+        <p></p>
+        <label>
+            City
+            <input type="text" th:field="*{city}">
+        </label>
+        <p></p>
+        <label>
+            Street
+            <input type="text" th:field="*{street}">
+        </label>
+        <p></p>
+        <label>
+            Building:
+            <input type="text" th:field="*{building}">
+        </label>
+        <p></p>
+        <label>
+            Apartment no.
+            <input type="text" th:field="*{apartment}">
+        </label>
+        <p></p>
+        <label>
+            Zip code
+            <input type="text" th:field="*{zipCode}">
+        </label>
+        <label>
+            Phone number
+            <input type="text" th:field="*{phoneNumber}">
+        </label>
+        <p></p>
+        <button type="submit">
+            Sign up
+        </button>
+    </form>
+</div>
+<p></p>
+<form method="get" th:action="@{/admin/clients}">
+    <button type="submit">Back to the client list</button>
+</form>
```

Comment:
Nie wykonujesz tutaj żadnej akcji - zamiast osobnego "formularza" i przycisku w nim, użyj linku (`<a th:href:"@{/admin/clients}">Back to the client list</a>`).

--- _side note:_
Przycisków używa się do wykonania interakcji z widokiem, do przekierowań, czy odniesień nawet w obrębie jednej podstrony (np. scrollujące do elementu o konkretnym id) należy wykorzystywać odnośniki (można je ostylować, żeby wyglądały jak przyciski, jednak w HTML powinien być użyty tag `<a>`). Takie podejście poprawia tzw. _accessibility_, jest zgodne ze specyfikacją HTML5 (semantyczne użycie tagów), a przy okazji zostanie jedna, zamiast 3 linijek kodu 😉

Reply:
Będę o tym pamiętać, dzięki :)